AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -118,8 +119,8 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
         gains_p[pol] = []
         pos = []
         stdv[pol] = []
-        h5.select(pol=pol,corrprods='cross',ants=active_ants)
-        h5.bls_lookup = calprocs.get_bls_lookup(active_ants,h5.corr_products)
+        h5.select(pol=pol,corrprods='cross',ants=h5.antlist,targets=[h5.catalogue.targets[TI] for TI in target_indices],compscans=compscan_index)
```

Comment:
This is not necessary, since `pol`, `corrprods` and `ants` won't affect time selections like `targets` and `compscans`.

Reply:
I think doing the full select  each time will hopefully prevent issues in the future.

Reply:
i separated this select into two in PR #463 , i am definitely in favour of not expanding it again now. +1 for the change suggested by ludwig.